### ISSUE TITLE
Set Date from JS only

### ISSUE THREAD

**Issue by nitin-jotwani** [charliekassel/vuejs-datepicker#443]
Is it possible to select a date from js directly without using model? Couldn't find any methods to achieve this.

**Comment by saintplay** [comment]
This seems like a **Vue** question

Sorry if I'm misunderstanding you, but have you tried using `value` prop?

**Comment by nitin-jotwani** [comment]
Yes, tried using value(). It does set date, but it does it on UI only, it doesn't bind to datepicker. Because as soon I open datepicker it gets to default again despite setting from value prop.

**Comment by jlsjonas** [comment]
@nitin-jotwani you can use `this.$refs.picker.setDate(new Date());` for that (where `picker` is the ref of the datepicker _html_ object)
